### ISSUE TITLE
[bug] error on liking without login is different from upstream

### ISSUE THREAD

**Issue by test2a** [stom79/TubeLab#126]
### Issue

on  the website it says "you need to be logged in to rate this video" if i try to like without logging in. the app instead says "you need to be authenticated to proceed to this action !!!" in a red error box.
we need to adjust this error according to the website to maintain consistency in wording and how alert is displayed

![Screenshot_20201202-114953](https://user-images.githubusercontent.com/39449563/100836161-d0ee1480-3494-11eb-9fe5-87be4fb573a4.png)
![Screenshot_20201202_115119](https://user-images.githubusercontent.com/39449563/100836199-e3684e00-3494-11eb-93a7-2b42e7f34566.png)

### Steps for reproducing the issue

---

Instance:

- [ ] TubeLab
- [ ] TubeAcad

Version of TubeLab:

Android version:

**Comment by stom79** [comment]
I understand, but I see two problems. I need to create a dedicated sentences for each action. Comments, thumbs up/down, Playlists, etc.
It makes more sentences to translate.

**Comment by test2a** [comment]
Nah. The website says you need to be logged in for both like and dislike. Thats all that needs to be done on this issue
